docs/Data_Submission_Portal/Users_Guide/Best_Practices.md

```diff
@@ -59,9 +59,10 @@ __Example 2:__ A 98 year old patient is diagnosed with cancer and dies three yea
 
 ## Array Submission
 
-Certain fields in the GDC, such as diagnosis.sites_of_involvement, are of type "array".  This allows multiple values to be submitted on one property. These values need to be uploaded in a `|`-delimited format. See the example below.  
+Certain fields in the GDC, such as diagnosis.sites_of_involvement, are of type "array".  This allows multiple values to be submitted on one property. These values need to be uploaded in a `|`-delimited format for TSV formatted uploads and a JSON-type array for JSON formatted uploads. See the example below.  
 
-__Example:__ `"sites_of_involvement" : "Kidney, Upper Pole|Kidney, Middle"`
+*  __Example (TSV):__ `Kidney, Upper Pole|Kidney, Middle`
```

Comment:
@wwysoc2, I think that we should mention that this value should go under the `sites_of_involvement` header for the TSV files? I think it would avoid any type of confusion that way.

Reply:
Thanks, @wwysoc2! Looks good to me.